etc/checkstyle/checkstyle.xml

```diff
@@ -2,195 +2,26 @@
 <!DOCTYPE module PUBLIC
 		"-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
 		"https://checkstyle.org/dtds/configuration_1_3.dtd">
-<module name="Checker">
-
+<module name="com.puppycrawl.tools.checkstyle.Checker">
 	<module name="SuppressionFilter">
-		<property name="file" value="${config_loc}/checkstyle-suppressions.xml"/>
+		<property name="file"
+		          value="${config_loc}/checkstyle-suppressions.xml" />
 	</module>
-
-	<!-- Root Checks -->
-	<module name="RegexpHeader">
-		<property name="headerFile" value="${config_loc}/checkstyle-header.txt"/>
-		<property name="fileExtensions" value="java"/>
+	<module name="com.puppycrawl.tools.checkstyle.checks.header.RegexpHeaderCheck">
+		<property name="headerFile" value="${config_loc}/checkstyle-header.txt" />
```

Comment:
According to this issue: https://github.com/spring-io/spring-javaformat/issues/303, our file must be this: https://github.com/spring-io/spring-javaformat/blob/main/spring-javaformat/spring-javaformat-checkstyle/src/main/resources/io/spring/javaformat/checkstyle/check/header-apache2.txt.
I wonder why it is not by default there in `SpringChecks`...
So, we would not need to manage that file, but rather rely in whatever Javaformat provides for us 🤷

Reply:
Actually it does that: https://github.com/spring-io/spring-javaformat/issues/70.

So, never mind: merging and moving on with that locally.

Will let you know with a fresh PR.

Thanks